src/locales/default.json

```diff
@@ -85,6 +85,16 @@
   "contectHash": "Content hash",
   "defaultSkin": "Default skin",
   "select": "Select",
+  "language": "Language",
+  "nativeLocales": {
```

Comment:
This is not something we want to translate in every language, this should live within another file like languages.json, we can use that file instead of this too: https://github.com/snapshot-labs/snapshot/pull/277/files#diff-84f11db7624ce9f3f4455f1d661c09c12559ac1b5ea595ab64e041632a8b2083R24